VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: Radar
VAL_TABLE_ Vt_ACC_SIG_Status 6 "Standstillwait" 5 "Standstill" 4 "Failure" 3 "Suspend" 2 "Active" 1 "Passive" 0 "Init" ;
VAL_TABLE_ vt_ACC_display_signal 3 "Check_Gear_position" 2 "Vehicle speed too low" 1 "Reserved" 0 "Reserved" ;


BO_ 336 ACC_status: 1 Vector__XXX
 SG_ ACC_sig_status : 2|3@1+ (1,0) [0|7] "" Vector__XXX
 SG_ ACC_disp_signal : 0|2@1+ (1,0) [0|3] "" Vector__XXX



EV_ env_tester_present_on_or_off: 0 [0|1] "" 0 1 DUMMY_NODE_VECTOR3 Vector__XXX;

EV_ env_variant_coading_done: 0 [0|1] "" 0 2 DUMMY_NODE_VECTOR3 Vector__XXX;

BA_DEF_  "MultiplexExtEnabled" ENUM  "No","Yes";
BA_DEF_  "BusType" STRING ;
BA_DEF_DEF_  "MultiplexExtEnabled" "No";
BA_DEF_DEF_  "BusType" "";
VAL_ 336 ACC_sig_status 6 "Standstillwait" 5 "Standstill" 4 "Failure" 3 "Suspend" 2 "Active" 1 "Passive" 0 "Init" ;
VAL_ 336 ACC_disp_signal 3 "Check_Gear_position" 2 "Vehicle speed too low" 1 "Reserved" 0 "Reserved" ;

